### DISCUSSION TITLE
IMGMOUNT - Support for .ins/.gog files broken with the latest 2024.03.01 release

### DISCUSSION THREAD

**Discussion by xvachon** [joncampbell123/dosbox-x#4877] [Q&A] [answered]
With the latest commits to IMGMOUNT, it seems that imgmount cannot mount .ins files anymore (essentially the .cue files for Good Old Games supported disk images). With the 2023.10.06 version, this command was working fine. 

![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/7642ac50-04fe-44d7-81cc-c080a1d2eac6)

If a syntax change is required, could you let me know?

Thank you!

**Comment by joncampbell123** [comment]
There's a list of file extensions it looks for with regard to ISO images. *.GOG is one of them. I've never heard of *.INS before.

**Comment by xvachon** [comment]
To clarify, .ins seems to behave like a .cue file. A default install for this DOS game include these files.
![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/b8657eb1-6507-4653-b58c-67379ba4c2cb)

The default dosbox 0.74 instructions for mounting look like this (first file). I modified the file paths so that they can launch properly with dosbox-X.
![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/c711d954-ede2-4c1d-9788-696e64a3e9d9)
![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/cf97a874-6ff5-4d72-9419-cfe377ffab33)

Prior to the 2024.03.01 update, I could launch the game without problems. After the update, somehow, I get an invalid file error.

Let me know if you require any extra instructions or debugging info.

**Comment by xvachon** [comment]
I retried to do the mounting described above with dosbox 0.74-2 and dosbox-x 2023.10.06 and it works
![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/de93081a-6399-4b4e-a5ef-ff965028b2bf)

![image](https://github.com/joncampbell123/dosbox-x/assets/16070349/35ee2374-c253-46af-960e-c2ca32a17a27)

**Comment by xvachon** [comment, marked as answer]
Closing answered per the commit in https://github.com/joncampbell123/dosbox-x/issues/4881
